client/tag/tagReducers.js

```diff
@@ -0,0 +1,22 @@
+import { TOGGLE_TAG } from './tagActions';
+
+const initialState = {
+  Romance: false,
+  Thrill: false,
+  Relaxation: false,
+  Food: false,
+  Family: false,
+  Outdoor: false,
+  Culture: false,
+  Landmark: false,
+};
+
+export default (state = initialState, action) => {
+  switch (action.type) {
+    case 'TOGGLE_TAG':
+      state[action.payload.tag] = !state[action.payload.tag];
```

Comment:
We should be creating a new state with the changed tag, and then return the new state object